app/models/survey.rb

```diff
@@ -0,0 +1,5 @@
+class Survey < ActiveRecord::Base
```

Comment:
Technically, controllers don't have to map directly to models. I would probably reserve the assessments controller to show a list of all assessments (index), show an individual one (show), create new ones (new), and edit existing ones (edit).

Rather, I like the idea of a SurveyController or something that allows someone to create _answers_ for all the questions. I guess it could be an AnswersController. Whatever you name it isn't super important, though.